google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/BigtableBatchingCallSettingsTest.java

```diff
@@ -38,7 +38,7 @@ public class BigtableBatchingCallSettingsTest {
       BatchingSettings.newBuilder()
           .setElementCountThreshold(10L)
           .setRequestByteThreshold(20L)
-          .setDelayThreshold(Duration.ofMillis(5))
```

Comment:
Im wondering if we should keep the test code the same so that we can verify these settings are ported to the new settings correctly? We can leave a todo to update them in a separate PR.